packages/cf.js/rollup.config.js

```diff
@@ -12,7 +12,8 @@ export default {
     },
     {
       file: pkg.iife,
-      name: "cfjs",
+      name: "window",
```

Comment:
Why name it `window`?

Reply:
Because we want `cf` to be exposed in `window`. Otherwise we'll get a double namespace.